### PR TITLE
zohorecruit connector modified according to API changes.

### PR DESCRIPTION
**Pull request by kesavany**

### CODE REVIEW

File: zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-jobOpenings/getAssociatedCandidates.xml [hunk 1]
```diff
@@ -1,24 +1,53 @@
...
+    <sequence>
+        <property name="uri.var.jobId" expression="$func:jobId"/>
+        <property name="uri.var.version" expression="$func:version"/>
+        <property name="uri.var.urlQuery" value=""/>
+        <script language="js">
+            <![CDATA[
+            var version = mc.getProperty('uri.var.version');
+            var urlQuery = "";
+            if (version != null && version != "") {
+               urlQuery +=  '&'+"version=" + version ;
```
Comment (Shakila) [inline zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-jobOpenings/getAssociatedCandidates.xml]: Format the code.

File: zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/zohorecruit/ZohorecruitConnectorIntegrationTest.java [hunk 2]
```diff
@@ -59,23 +59,25 @@ public void setEnvironment() throws Exception {
...
     	esbRequestHeadersMap.put("Action", "urn:addRecords");
         RestResponse<JSONObject> esbRestResponse =
                 sendJsonRestRequest(proxyUrl, "POST", esbRequestHeadersMap, "esb_addRecords_mandatory.json");
         
         String recordId =
                 esbRestResponse.getBody().getJSONObject("response").getJSONObject("result")
-                        .getJSONObject("recorddetail").getJSONObject("FL").get("content").toString();
-        connectorProperties.put("recordId", recordId);
+                        .getJSONObject("recorddetail").getJSONArray("FL").getJSONObject(0).getString("content");
+        connectorProperties.put("recordId1", recordId);
```
Comment (Shakila) [inline zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/zohorecruit/ZohorecruitConnectorIntegrationTest.java]: Use proper name for recordId1.

Comment (kesavany) [inline zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/zohorecruit/ZohorecruitConnectorIntegrationTest.java]: Fixed in https://github.com/wso2/esb-connectors/pull/406

File: zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-jobOpenings/getAssociatedCandidates.xml (hunk 1 above)
Comment (kesavany) [inline zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-jobOpenings/getAssociatedCandidates.xml]: Fixed in https://github.com/wso2/esb-connectors/pull/406

File: zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-candidates/changeStatus.xml [hunk 3]
```diff
@@ -1,71 +1,68 @@
...
+            <![CDATA[
+            var jobId = mc.getProperty('uri.var.jobId');
+            var comments = mc.getProperty('uri.var.comments');
+            var version = mc.getProperty('uri.var.version');
+            var urlQuery = "";
+            if (jobId != null && jobId != "") {
+               urlQuery += "jobId=" + jobId + '&';
+            }
+            if (comments != null && comments != "") {
+               urlQuery += "comments=" + comments + '&';
```
Comment (Shakila) [inline zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-candidates/changeStatus.xml]: Use 4 spaces for indentation

Comment (kesavany) [inline zohorecruit/zohorecruit-connector/zohorecruit-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/zohorecruit-candidates/changeStatus.xml]: Fixed in https://github.com/wso2/esb-connectors/pull/406

**Pull request merged**